exporter/datasetexporter/logs_exporter_stress_test.go

```diff
@@ -0,0 +1,165 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build integration
+// +build integration
+
+package datasetexporter
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+	"go.opentelemetry.io/collector/exporter/exportertest"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+)
+
+type SuiteLogsE2EExporter struct {
+	suite.Suite
+}
+
+func (s *SuiteLogsE2EExporter) SetupTest() {
+	os.Clearenv()
+}
+
+func TestSuiteLogsE2EExporterIntegration(t *testing.T) {
+	suite.Run(t, new(SuiteLogsE2EExporter))
+}
+
+func (s *SuiteLogsE2EExporter) TestConsumeLogsManyLogsShouldSucceed() {
+	const maxDelay = 200 * time.Millisecond
+	createSettings := exportertest.NewNopCreateSettings()
+
+	const MaxBatchCount = 20
+	const LogsPerBatch = 10000
+	const ExpectedLogs = uint64(MaxBatchCount * LogsPerBatch)
```

Comment:
not sure why these are declared as const and exported, 
```suggestion
	expectedLogs := uint64(maxBatchCount * logsPerBatch)
```

Reply:
Sorry! I am `STILL_NOT_USED` to `lowerCasedConstants`.

Reply:
But they are inside function => so they will not be exported. I would say -> but I will fix it anyways.

Reply:
Fixed